### PR TITLE
[2/N][Refactor] Run pre-commit for all files (without golangci-lint)

### PR DESCRIPTION
**Pull request by MortalHappiness**
## Why are these changes needed?

Run pre-commit for all files (without the new golangci-lint rules in https://github.com/ray-project/kuberay/pull/2128)

Depends on https://github.com/ray-project/kuberay/pull/2127

## Related issue number

N/A

## Checks

- [x] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [x] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: docs/reference/api.md [hunk 1]
```diff
@@ -477,5 +477,3 @@ _Appears in:_
 | `rayStartParams` _object (keys:string, values:string)_ | RayStartParams are the params of the start command: address, object-store-memory, ... |
 | `template` _[PodTemplateSpec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#podtemplatespec-v1-core)_ | Template is a pod template for the worker |
 | `scaleStrategy` _[ScaleStrategy](#scalestrategy)_ | ScaleStrategy defines which pods to remove |
-
```
Comment (kevin85421) [inline docs/reference/api.md:480]: This is a generated file. Perhaps we should not lint it.

Comment (MortalHappiness) [reply docs/reference/api.md]: Done

**Review by kevin85421 — COMMENTED** [review]
Files under `config/grafana` are generated files. It is unnecessary to update them.

**Comment by MortalHappiness** [comment]
> Files under `config/grafana` are generated files. It is unnecessary to update them.

Done

**Review by kevin85421 — APPROVED** [review]
Nice!

**Pull request merged**